src/bin/keycloakify/generateFtl/generateFtl.ts

```diff
@@ -13,6 +13,7 @@ import { Reflect } from "tsafe/Reflect";
 // https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/forms/login/freemarker/Templates.java
 export const pageIds = [
     "login.ftl",
+    "login-username.ftl",
```

Comment:
These didn't appear in any particular order. Is this an okay place to put this?

Reply:
sure